app/src/components/common/footer/FooterInfo.js

```diff
@@ -12,7 +12,7 @@ export default function FooterInfo() {
             chaletinteriordesign@gmail.com
           </p>
         </div>
-        <div className="">
+        <div>
```

Comment:
please check all empty divs. some are not necessary

Reply:
well noted, done!